lib/typescript/httpclient/model/Message.ts

```diff
@@ -1,13 +1,6 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const camelcaseKeys = require('camelcase-keys');
-
-export enum Source {
-  facebook = 'facebook',
-  google = 'google',
-  chatplugin = 'chatplugin',
-  smsTwilio = 'twilio.sms',
-  whatsappTwilio = 'twilio.whatsapp',
-}
```

Comment:
the enum SourceType was duplicated in both Message and Channels in the httpClient library. I've kept the one in Channels.

Reply:
source was a better name :D (sorry... I suggested source type somewhere else.... without realising we call the field source everywhere in the api)